eng/pipelines/libraries/helix-queues-setup.yml

```diff
@@ -90,6 +90,7 @@ jobs:
     # OSX arm64
     - ${{ if eq(parameters.platform, 'OSX_arm64') }}:
       - OSX.1100.ARM64.Open
+      - OSX.1200.ARM64.Open
```

Comment:
Just curious, how can I tell this is libraries tests, as opposed eg to CLR tests ? It's not clear to me - I'm not familiar with how this works.

Reply:
Ad far as I understand
CLR: eng/pipelines/coreclr/templates/helix-queues-setup.yml
Libs: eng/pipelines/libraries/helix-queues-setup.yml 

But it is still black box for me how it is orchestrated.

@ViktorHofer is there any docs available?

Reply:
This file associates the queue to platform. To see if it runs, you'll have to check the runtime ymls to see if the platform is defined.  A quick spot check looks like osx-arm64 is defined only for the runtime tests.